pyproject.toml

```diff
@@ -83,7 +83,7 @@ docs = [
     "nbconvert == 6.5.0",
     "ipython == 8.4.0",
     "jupyter == 1.0.0",
-    "jupyter-client == 7.4.9",
+    "jupyter-client >= 8.0.2",
```

Comment:
Should we just remove this and let the latest version get installed since we don't explicitly use this anywhere in the codebase?